docs/developer-docs/multi-chain/ethereum/using-eth/eth-comparison.mdx

```diff
@@ -0,0 +1,124 @@
+
+<table>
+<thead>
+<tr>
+  <th> Ethereum term </th>
+  <th> ICP term </th>
+</tr>
+</thead>
+
+<tbody>
+<tr>
+  <td> Smart contract </td>
+  <td> Canister </td>
+</tr>
+
+<tr>
+  <td> Gas </td>
+  <td> Cycles </td>
+</tr>
+
+<tr>
+  <td> Shard </td>
+  <td> Subnet </td>
+</tr>
+
+<tr>
+  <td> Node </td>
+  <td> Replica </td>
+</tr>
+
+<tr>
+  <td> Account </td>
+  <td> Principal </td>
+</tr>
+
+<tr>
+  <td> DAO </td>
+  <td> SNS, a DAO using ICP's [unique framework](/docs/current/developer-docs/daos/sns/). However, ICP DAOs can launch without using this framework. </td>
+</tr>
+</tbody>
+</table>
+
+In addition, ICP has implemented similar token standards as Ethereum:
+
+<table>
+<thead>
+<tr>
+  <th> Ethereum standard </th>
+  <th> ICP standard </th>
+</tr>
+</thead>
+
+<tbody>
+<tr>
+  <td> ERC-20 </td>
+  <td> [ICRC-1/ICRC-2](../../../../references/icrc1-standard) </td>
+</tr>
+
+<tr>
+  <td> ERC-721 </td>
+  <td> [ICRC-7](https://github.com/dfinity/ICRC/blob/icrc7-wg-draft/ICRCs/ICRC-7/ICRC-7.md) </td>
+</tr>
+</tbody>
+</table>
+
+## Smart contract development comparison
+
+Both Ethereum and ICP allow developers to deploy programs that run on the blockchain. These programs are referred to as smart contracts in Ethereum and [canisters](/docs/current/developer-docs/smart-contracts/overview/introduction) on ICP. Occasionally, the phrase “canister smart contracts” is also used on ICP.
+
+This comparison provides an overview of the key differences in smart contract development on Ethereum and ICP.
+
+## Key differences:
+
+### Creation:
+**Ethereum**: Smart contracts are written in Solidity and deployed immutably onto the blockchain.
+**ICP**: Canisters can be written in any language that complies to WASM including [Rust](/docs/current/developer-docs/backend/rust/), [Typescript](/docs/current/developer-docs/backend/Typescript), [Motoko](/docs/current/motoko/main/getting-started/motoko-introduction), and [Python](/docs/current/developer-docs/backend/python/).
+
+### Deployment
+**Ethereum**: Developers typically use development frameworks like Hardhat or Truffle to compile and deploy their smart contracts. 
+**ICP**: Developers use dfx which is a similar development environment. [DFX](/docs/current/developer-docs/getting-started/install/#installing-dfx-via-dfxvm) is a command-line tool that interacts with the IC SDK and allows developers to compile and deploy their canisters.
+
+### Testing
+
+**Ethereum**: Developers can utilize testnets such as Sepolia for deploying and testing smart contracts before deploying to the mainnet.
+**ICP**: Currently, testing is done in the local environment. A [playground environment](/docs/current/developer-docs/developer-tools/ide/playground/) on mainnet is available for canisters, but deployment time is limited to 20 minutes. However, the ability to upgrade and delete canisters on the mainnet allows developers greater flexibility to iterate and refine their canisters.
```

Comment:
```suggestion
**ICP**: Currently, testing is done in the local environment. A [playground environment](/docs/current/developer-docs/developer-tools/ide/playground/) on mainnet is available for canisters, but deployment time is limited to 20 minutes. However, the ability to upgrade and delete canisters on the mainnet allows developers greater flexibility thank on Ethereum to iterate and refine their canisters.
```

Reply:
@jessiemongeon1 Can you please take a look at this suggestion?

Reply:
Updated suggestion